stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/ChatApi.kt

```diff
@@ -273,6 +273,7 @@ internal interface ChatApi {
         channelType: String,
         channelId: String,
         messageId: String = "",
+        threadId: String = "",
```

Comment:
Previously, this method accept empty `messageId` to  mark the full channel as read, in another case mark the channel read until the messageId.

Now that it contains an extra argument (`threadId`) it is a bit confusing what it will does if, for example, we pass a messageId and a threadId...

It would be better to have a new `markThreadRead()` and then extract common logic on the `MoshiChatApi` to reuse whatever is needed